lib/site_prism/element_container.rb

```diff
@@ -131,6 +133,18 @@ def create_waiter(element_name, *find_args)
       end
     end
 
+    def create_disappearance_waiter(element_name, *find_args)
+      method_name = "wait_for_#{element_name}_disappearing"
```

Comment:
Because sometimes i got follow error: 
`Failure/Error: organization_page.wait_until_sharing_settings_popup_invisible 
     Selenium::WebDriver::Error::StaleElementReferenceError:
       stale element reference: element is not attached to the page document
         (Session info: chrome=58.0.3029.81)`